cleverpush/src/main/java/com/cleverpush/util/VersionComparator.java

```diff
@@ -76,15 +76,15 @@ public int compare(Object o1, Object o2) {
                 suffix2 = tokenizer2.getSuffix();
                 if (number2 != 0 || suffix2.length() != 0) {
                     // Version one is longer than version two, and non-zero
-                    return -1;
+                    return VersionComparison.LESS_THAN;
                 }
             }
             while (tokenizer2.MoveNext());
 
             // Version two is longer than version one, but zero
-            return 0;
+            return VersionComparison.EQUALS;
         }
-        return 0;
+        return VersionComparison.EQUALS;
```

Comment:
not sure if Equals is correct here, can you give some examples here?

Reply:
here if we get the 0 from this method it means that both the versions are Equal.